unfetter-discover/unfetter-store
# added working docker test command

fix the travis build error.  this is not the cleanest solution, but works for now

## To Test

* Pull this branch
* `cd unfetter-ctf-ingest/`
*  build docker `docker build -t ctf-ingest2 .`
* run test from with in docker `docker run -it ctf-ingest2 npm run docker:test`
* verify all specs pass
* clean your images `docker rmi ctf-ingest2`
* also this pr should not show the Typescript compile error
